### PR TITLE
C11 mutex

### PR DESCRIPTION
**Pull request by mbattista**
C11 changes for the gtk module

### CODE REVIEW

File: modules/gtk/call_window.c [hunk 1]
```diff
@@ -396,6 +395,9 @@ struct call_window *call_window_new(struct call *call, struct gtk_mod *mod,
 	GtkWidget *button_box, *vbox, *hbox;
 	GtkWidget *duration;
 	int err = 0;
+	err = mtx_init(&last_data_mut, mtx_plain) != thrd_success;
+	if (err)
```
Comment (alfredh) [inline modules/gtk/call_window.c:399]: please set `err` to `ENOMEM` here

Comment (alfredh) [reply modules/gtk/call_window.c:399]: sorry, I should have been more clear..

curly brackets are missing. Example code here:

https://github.com/baresip/baresip/blob/main/modules/coreaudio/player.c#L106

Comment (mbattista) [reply modules/gtk/call_window.c:399]: Saw the example code when I looked for other occurrences of ENOMEN.

Thanks for finding the missing curly brackets. I am a little embarrassed, that this really just happened.

**Comment by mbattista** [comment]
should I force push a single commit with all the changes?

**Comment by alfredh** [comment]
thank you

it is fine to push a bunch of commits. When merging I do "squash and merge" ...

**Pull request merged**